Resources/public/templates/loginform.hbt

```diff
@@ -1,23 +1,25 @@
 <div class="ez-loginform-content">
     <form action="" class="ez-loginform" method="post">
-        <h1 class="ez-loginform-title"><img class="ez-logo" src="{{ asset 'bundles/ezplatformui/img/logo.png' }}" alt="eZ Logo"/> eZ Platform</h1>
+        <h1 class="ez-loginform-title"><img class="ez-logo" src="{{ asset 'bundles/ezplatformui/img/logo.png' }}" alt="eZ Logo"/>{{translate 'loginform.ezplatform' 'login'}}</h1>
```

Comment:
not sure we want to translate *eZ Platform* but I'm sure *eZ Logo* should be translated